packages/app/src/vis-packs/core/line/MappedLineVis.tsx

```diff
@@ -104,10 +106,31 @@ function MappedLineVis(props: Props) {
   const visDomain = useVisDomain(customDomain, combinedDomain);
   const [safeDomain] = useSafeDomain(visDomain, combinedDomain, yScaleType);
 
+  const selection = getSliceSelection(dimMapping);
   const xDimIndex = dimMapping.indexOf('x');
 
-  const { getExportURL } = useDataContext();
-  const selection = getSliceSelection(dimMapping);
+  const abscissaParams: AxisParams = {
+    label: axisLabels[xDimIndex],
+    value: numAxisArrays[xDimIndex],
+    scaleType: xScaleType,
+  };
+
+  const auxiliaries = auxArrays.map((array, i) => ({
+    label: auxLabels[i],
+    array,
+    errors: auxErrorsArrays[i],
+  }));
+
+  const exportEntries = useExportEntries(['npy', 'csv'], dataset, selection, {
+    csv: () =>
+      generateCsv(
+        valueLabel,
+        dataArray,
+        abscissaParams,
+        errorsArray,
+        auxiliaries,
+      ),
```

Comment:
This is the entry point for _NX Line_ CSV exports. As mentioned, it's not perfect because I don't have easy access to the axis/auxiliary dataset names; I only have access to their label (which can include `long_name` and `units`). I will work on this.